src/scales/scale.time.js

```diff
@@ -177,7 +177,7 @@ function interpolate(table, skey, sval, tkey) {
 	return prev[tkey] + offset;
 }
 
-function toTimestamp(input, options) {
+function toTimestamp(input, adapter, options) {
```

Comment:
I would try to keep the same logic for all these methods: either pass `adapter` as the first argument or the last one, but not randomly positioned. Since it's not an optional argument, I would vote to have it always as first argument.

Reply:
What do you think about making these as private class members of the scale so that we don't need to pass in the adapter at all? It'd probably save us from having to pass in the options, scale, min/max, in a few other places as well

Reply:
We could but unsure it would be better. Though, I would leave this for another PR, if really needed, to avoid introducing too many changes in this one and stay focused on the introduced feature. For now, I would maybe move `adapter` as first argument. Or maybe better, pass `scale` instead, so for example this method would become:

```javascript
function toTimestamp(scale, input) {
    var options = scale.options.time;
    var adatper = scale._adapter;
    // ...
}
```

What do you think?

Reply:
great idea! done